drivers/mtd/mtd_nvs.h

```diff
@@ -0,0 +1,102 @@
+/****************************************************************************
+ * drivers/mtd/mtd_nvs.h
```

Comment:
could merge mtd_nvs.h into mtd_nvs.c, since all info is the internal detail which doesn't need share with other party.